### DISCUSSION TITLE
Select date time difference with raw sql vs supabase js

### DISCUSSION THREAD

**Discussion by JamesWong1999** [supabase/supabase#4875] [Questions] [answered]
Hi there,

I am confused why the supabaseJS return correct date time and include the time zone,

SupabaseJS -> `2022-01-08T08:56:31.463+00:00`

but when i write a function the date time return is in different format and the time zone display in the app is incorrect as well.

Raw query -> `2022-01-08T08:56:31.463`

I must have miss something important here, please give a hint. 

thank you.

**Comment by steve-chavez** [comment, marked as answer]
supabase-js transforms the timestamptz to json under the hood.

```sql
set timezone to utc;

select now();
              now
-------------------------------
 2022-01-08 18:46:00.267152+00

select to_json(now());
              to_json
------------------------------------
 "2022-01-08T18:46:02.032392+00:00"
```

**Reply by JamesWong1999** [reply]
thank you very much @steve-chavez
